apps/cli/src/commands/project/create.ts

```diff
@@ -0,0 +1,134 @@
+import { intro, isCancel, outro, select, spinner, text } from "@clack/prompts";
+import { Args, Command, Flags } from "@oclif/core";
+import axios, { AxiosResponse } from "axios";
+import { blue, bold, cyan, grey, red, underline } from "kleur/colors";
+import { triggerCancel } from "../../lib/helpers";
+import { getCliConfigFromKeyStore } from "../../lib/keyStore";
+
+type NewProjectResponse = {
+  name: string;
+  slug: string;
+  createdAt: Date;
+};
+
+const loader = spinner();
+const API_BASE = process.env.API_BASE || `http://localhost:3000`;
+const LINKS = {
+  projectsPage: `${API_BASE}/projects`,
+  documentation: `${API_BASE}/docs/cli/projects`,
+};
+
+export default class ProjectCreate extends Command {
+  static description =
+    "Create a new project in the Envles. This command will create a new project with the specified name and add it to the list of projects in the Envless project. If no name is provided, You'll be prompted to provide one";
+
+  static examples = [
+    `
+      $ envless project create
+      $ envless project create {project name} {project slug}
+      `,
+  ];
+
+  static args = {
+    name: Args.string({
+      description: `This is the project's name`,
+      dependsOn: ["slug"],
+    }),
+    slug: Args.string({
+      description: `This will be used as the slug for the project`,
+      dependsOn: ["name"],
+    }),
```

Comment:
Lets go with flags instead of args, it'll be too long and orders are hard to define when passing args. For reference:

https://github.com/envless/envless/blob/e31a312662c16093de7e34c12d6a4980479bed2e/apps/cli/src/commands/project/link.ts#L23-L33